src/ui/i18n/translations/common_fr.json

```diff
@@ -17,5 +17,8 @@
     "accessGovernance": "Accéder à la gouvernance"
   },
   "filters": "Filtres",
-  "dataverse": "Dataverse"
+  "dataverse": "Dataverse",
+  "from": "Depuis le",
+  "to": "Jusqu'au",
+  "colon": "\u00A0:\u00A0"
```

Comment:
I'm not sure it's a good practise to deal with unicode character here...

Reply:
I understand the concern about using unicode characters, using plain text is more readable. We can replace the Unicode with regular spaces if necessary but this may not provide the same control over spacing and line breaks. 

We can do this: `"colon": " : "`
Or add `&nbsp;` in the component but we would need to do it each time we have to deal with colons.

What do you think?

Reply:
I think you can manage space directly in translations files, without any unicode. I made it in my actual PR if you want to check !

Reply:
Done in [c364741](https://github.com/okp4/dataverse-portal/pull/175/commits/c364741e2a5cb183a5112829b0cfcbf9c1ca2d49)!